formatTest/unit_tests/expected_output/infix.re

```diff
@@ -948,3 +957,49 @@ let containingObject = {
       "newvalue" [@shouldBeRenderedOnString]
   }
 };
+
+let x = foo |> z;
+
+let x =
+  foo
+  |> f
+  |> g;
+
+let x =
+  foo
+  |> somelongfunctionname "foo"
+  |> anotherlongfunctionname "bar" 1
+  |> somelongfunction
+  |> bazasdasdad;
+
+let code =
+  JSCodegen.Code.(
+    create
+    |> lines
+         Requires.(
+           create
+           |> import_type
+                local::"Set" source::"Set"
```

Comment:
Because of the printing width of the tests.
See Example section in my PR description for the default printing width.